server/delay_server/src/irodsDelayServer.cpp

```diff
@@ -635,8 +635,11 @@ int main(int argc, char** argv)
         try {
             return irods::get_advanced_setting<const int>(irods::CFG_DELAY_SERVER_SLEEP_TIME_IN_SECONDS);
         }
-        catch (const irods::exception& e) {
-            logger::delay_server::error(e.what());
+        catch (...) {
+            logger::delay_server::warn("Could not retrieve [{}] from advanced settings configuration. "
+                                       "Delay server will sleep for {} seconds.",
```

Comment:
The message printed in `msiSetReServerNumProc` mentions the "default value" being used. Any thoughts on mentioning that in the rest of these messages as well?

Reply:
agreed

Reply:
Oh, you're talking about stating that the fallback value is the default value.

Sure. Will update.

Reply:
done.